packages/grid/x-grid-modules/src/components/error-message.tsx

```diff
@@ -0,0 +1,6 @@
+import * as React from 'react';
+import { GridOverlay } from './styled-wrappers/window-overlay';
+
+export function ErrorMessage({ message }: { message?: string }) {
```

Comment:
Only a coding style around React components:

```suggestion
interface ErrorMessageProps {
  message?: string;
}

export function ErrorMessage(props: ErrorMessageProps) {
  const { message } = props;
```